### PR TITLE
Fix (_WD_Alert): Improve alert detection

### PR DESCRIPTION
**Pull request by Danp2**

### CODE REVIEW

**Comment by Danp2** [comment]
@mlipok Please review and test this change for proper functionality.

File: wd_core.au3 [hunk 1]
```diff
@@ -888,7 +888,7 @@ EndFunc   ;==>_WD_ExecuteScript
 ;                  $sOption  - [optional] a string value. Default is ""
 ; Return values .: Success - Requested data returned by web driver.
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
-;                  - $_WD_ERROR_Exception
+;                  - $_WD_ERROR_NoAlert
```
Comment (mlipok) [inline wd_core.au3:891]: $_WD_ERROR_NoAlert --- Where this is set?

Comment (Danp2) [reply wd_core.au3:891]: __WD_DetectError, which gets called by __WD_Get, __WD_Post, etc.

Comment (mlipok) [reply wd_core.au3:891]: aa...

**Review by mlipok — APPROVED** [review]
Looks and works good..

**Pull request merged**